fhir-server/liberty-config/configDropins/disabled/datasource-proxy.xml

```diff
@@ -0,0 +1,12 @@
+<server>
+    <library id="fhirSharedLib">
+        <fileset dir="${shared.resource.dir}/lib/derby" includes="*.jar"/>
+        <fileset dir="${shared.resource.dir}/lib/db2" includes="*.jar"/>
+        <fileset dir="${shared.resource.dir}/lib/postgresql" includes="*.jar"/>
+        <fileset dir="${shared.resource.dir}/lib/fhir" includes="*.jar"/>
```

Comment:
Remove this line?

Reply:
```suggestion
```

Reply:
Actually, no.  The FHIR lib needs to be there in order to use the FHIRProxyXADataSource. You are right that we're not actually packaging these libs here though, so I'm not really sure the best way to handle it.  Maybe just add a comment that says you need to add these here to enable it?
```suggestion
        <!-- Copy the fhir-persisence-proxy jar and its dependencies to this dir to enable it -->
        <fileset dir="${shared.resource.dir}/lib/fhir" includes="*.jar"/>
```

Reply:
Sounds good.